st2tests/st2tests/config.py

```diff
@@ -484,6 +484,16 @@ def _register_garbage_collector_opts():
             default=None,
             help="Trigger instances older than this value (days) will be automatically deleted.",
         ),
+        cfg.IntOpt(
+            "rule_enforcement_ttl",
+            default=None,
+            help="Rule enforcements older than this value (days) will be automatically deleted.",
```

Comment:
May be good to add something along the lines  of "Defaults to None aka disabled by default." / "If set to None, it's disabled." to the end of the help strings.